### PR TITLE
Resolve #20. Rebuild the code.

### LINKED ISSUE #20: When there is something in the terminal, the commands that are run will be superimposed directly.
I write "Hello world" in the terminal, and don't clear them. Then I run my cpp file, the command will be "Hello worlda.exe".
To solve it, please call a command to clear the terminal first.

### PR DESCRIPTION
**Pull request by Anti-Li**

### CODE REVIEW

**Comment by Anti-Li** [comment]
The escape sequence can clear the line.

**Comment by danielpinto8zz6** [comment]
no
`            await new Promise(resolve => setTimeout(resolve, 700)); // await for a few time.
`
why wait?

**Comment by danielpinto8zz6** [comment]
Another thing, why make terminal functions static?

File: src/VSCodeUI.ts [hunk 1]
```diff
@@ -23,11 +24,25 @@ export namespace VSCodeUI {
...
-        public runInTerminal(command: string, options?: ITerminalOptions): void {
-            const defaultOptions: ITerminalOptions = { addNewLine: true, name: "Compile Run" };
-            const { addNewLine, name, cwd } = Object.assign(defaultOptions, options);
+        static get clearLineChar() {
+            return process.platform === "win32" ? '\r' : '\r\0' + '33[k';
+        }
+
+        static async runInTerminal(command: string) {
+            CodeTerminal.runInTerminal(this.clearLineChar, this.defaultOptions);
+            await new Promise(resolve => setTimeout(resolve, 700)); // await for a few time.
```
[the remark this answers is not in the document]
Comment (danielpinto8zz6) [reply src/VSCodeUI.ts]: Thats because you made runInTerminal async...

[the remark this answers is not in the document]
Comment (Anti-Li) [reply src/VSCodeUI.ts]: Really? I think function "sendText" doesn't delay, but the command working does delay **in terminal**(like cmd). That why I make a wait.

**Pull request closed without merging**